lib/email_tracking_retriever.py

```diff
@@ -285,10 +287,20 @@ def clean_email_content(email_str) -> str:
   return email_str
 
 
-def log_in_if_necessary(driver: WebDriver):
+def log_in_if_necessary(driver: WebDriver, amazon_config):
   driver.get('https://www.amazon.com/gp/your-account/order-history/ref=ppx_yo_dt_b_orders')
   orders_containers = driver.find_elements_by_id('ordersContainer')
   if len(orders_containers) == 0:
     # likely a login screen
-    driver.find_element_by_css_selector('form[name="signIn"]')
-    input('Please log in to an Amazon account on the opened Chrome profile. Hit ENTER when done.')
+    if "email" in amazon_config and amazon_config["email"]:
+      print("Signing into Amazon ...")
+      driver.find_element_by_css_selector('form[name="signIn"]')
+      driver.find_element_by_css_selector('input[type="email"]').send_keys(amazon_config['email'])
+      driver.find_element_by_css_selector('input[type="submit"]').click()
+      driver.find_element_by_css_selector('input[type="password"]').send_keys(amazon_config['password'])
+      driver.find_element_by_css_selector('input[type="submit"]').click()
+
+      if (amazon_config['twoFactor']):
```

Comment:
let's be a bit more automatic about this; Amazon might still ask for a OTP even if you don't have two factor auth. Try looking for something that should appear on the page post-successful-login and if that doesn't show up, then ask the user to enter the OTP (we can remove that config point too)